### ISSUE TITLE
About favicon display error in bookmark

### ISSUE THREAD

**Issue by NightSharp** [piroor/treestyletab#2536]
When I bookmark tabs which are used to be disgarded by Auto Tab Disgard,
The favicon are always different from what they should be.

> **I've customized the style for discarded tabs' favicon in TST, and ATD alwals added a little gray dot on the bottom right of them when those tabs were disgarded by ATD.**

These tabs' favicon displayed in Bookmarks are always still in the style what I quoted above, though they are loaded normally and their favicons are displayed in TST normally now.
![the style compare](https://user-images.githubusercontent.com/9748063/77839558-66bbfc00-71b0-11ea-95cc-d8c4c05a34aa.png)In TST (Not discarded by ATD)
![the example](https://user-images.githubusercontent.com/9748063/77839512-fb722a00-71af-11ea-8f07-a1322b592df1.png)In Bookmarks (The last one are used to be discarded by ATD)

**Comment by piroor** [comment]
It is a restriction of WebExtensions API itself. We cannot specify favicon URL for a bookmark created via API.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/create
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/CreateDetails

**Comment by NightSharp** [comment]
> 
> 
> It is a restriction of WebExtensions API itself. We cannot specify favicon URL for a bookmark created via API.
> https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/create
> https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/CreateDetails

@piroor  Now I find out that the problem  appears in only one url. I don't know what's going on...

**Comment by piroor** [comment]
Is the URL accessible for me? Then could you put it here?

**Comment by NightSharp** [comment]
> 
> 
> Is the URL accessible for me? Then could you put it here?

@piroor  Here it is.
https://www.bilibili.com/video/av94548443

**Comment by irvinm** [comment]
@piroor does the URL help you?

**Comment by piroor** [comment]
Sorry I forgot to check the URL.

I've tested the URL and realized that "bookmark with favicon having gray circle" problem happens without TST. This is the screenshot I've tried with Nightly and Auto Tab Discard:
![image](https://user-images.githubusercontent.com/70062/81495700-170f3b00-92ed-11ea-8067-34e365fc1a39.png)
There are two more tabs and both are bookmarked to the bookmarks toolbar. "Firefox Privacy Notice" has its correct favicon,  but bilibili's bookmark has discarded favicon - it should not have a gray circle.

TST's sidebar just mirrors favicons from Firefox's native tabs. You'll see same favicon on the TST sidebar and Firefox's native tab bar in the screenshot below:
![image](https://user-images.githubusercontent.com/70062/81495777-bc2a1380-92ed-11ea-8684-77e5c55538c9.png)

Tab favicons in these screenshots are completely expected, except the bookmark for the bilibili's page has a gray circle. So I think this is not a problem of TST, but a problem of Firefox itself or Auto Tab Discard.
